frontend/src/metabase/public/components/widgets/LegaleseStep/LegaleseStep.styled.tsx

```diff
@@ -0,0 +1,10 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { Stack } from "metabase/ui";
+
+export const LegaleseStepDetailsContainer = styled(Stack)`
+  ${({ theme }) => css`
+    border: 1px solid ${theme.colors.border[0]};
+    border-radius: ${theme.radius.md};
+  `}
```

Comment:
```suggestion
  border: 1px solid ${({ theme }) => theme.colors.border[0]};
  border-radius: ${({ theme }) => theme.radius.md};
```

I haven't found other places where we do `${({ theme }) => css`

(If you accept this, we'll need to remove the css import too)

Reply:
Hmm, I'm not sure I like this change. What's the benefit over the original? I like my solution since it keeps the number of arrow functions to a minimum, but if you have a better reason I'd be happy to change it

Reply:
It's mostly for consistency with how we usually do it, I checked emotion's docs and I haven't find any example of using `css` inside a styled like that.
The docs mostly use tagged literals as in my suggestions with inline interpolation, or use the object syntax like this:
```tsx
const Container = styled.div(props => ({
  display: 'flex',
  flexDirection: props.column && 'column'
}))
```

Anyway, this is a nit/personal opinion so feel free to ignore it

Reply:
Makes sense. I have a bias towards consistency over what makes me feel good so I'll change it 🙂

Reply:
Both look fine IMO. But if you have more than a handful of properties that need `theme`, I'd be biased toward @oisincoveney 's solution more because we don't have to repeat `theme` every time.

Reply:
Ok, I'll leave it as is for now then